src/Behat/Symfony2Extension/Context/Argument/ServiceArgumentResolver.php

```diff
@@ -74,9 +74,13 @@ private function resolveArgument($argument)
             return $service;
         }
 
-        $withParameters = $this->replaceParameters($container, $argument);
+        $resolvedParam = $this->replaceParameters($container, $argument);
 
-        return $this->escape($withParameters);
+        if (!is_string($resolvedParam)) {
```

Comment:
I can use the ternary operator, but could you tell me the benefit in this case please?